shop/models/fields.py

```diff
@@ -12,6 +12,8 @@
     import psycopg2
 
     version = re.search('([0-9.]+)', psycopg2.__version__ or "0").group(0)
+    # To be able to use the Django version of JSONField, it requires to have PostgreSQL ≥ 9.4 , otherwise some issues
+    #  could be faced.
     if POSTGRES_FLAG and (LooseVersion(version) >= LooseVersion('2.5.4')):
```

Comment:
Yes. Actually the new jsonfield shipped with django required postgres >=
9.4 and psycopy2 >= 2.5.4 .since I haven't found a way to check
postgresql's version ,I plan to add it in the documentation.
https://docs.djangoproject.com/en/1.10/ref/contrib/postgres/fields/#jsonfield

On Sep 6, 2016 7:21 PM, "René Fleschenberg" <notifications@github.com>
wrote:

> In shop/models/fields.py
> <https://github.com/awesto/django-shop/pull/399#discussion_r77690593>:
>
> > @@ -12,6 +12,8 @@
> >      import psycopg2
> >
> >      version = re.search('([0-9.]+)', psycopg2.__version__ or "0").group(0)
> > +    # To be able to use the Django version of JSONField, it requires to have PostgreSQL ≥ 9.4 , otherwise some issues
> > +    #  could be faced.
> >      if POSTGRES_FLAG and (LooseVersion(version) >= LooseVersion('2.5.4')):
>
> This looks like it checks the psycopg version, not the version of Postgres
> itself. Or are these two somehow related?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/awesto/django-shop/pull/399/files/19dc04eb48a9484540298aa9a15fca016486921b..57fc812d5cb3e8d08d24a9f2c54954b8c7a0b8f7#r77690593>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADTfRYEdSVA_s-nstRI9wWvrxRpXU0NZks5qna80gaJpZM4Ju_OR>
> .
>